testplan/exporters/testing/pdf/renderers/entries/assertions.py

```diff
@@ -697,14 +701,18 @@ def get_matched_row_data(
             include_columns = include_columns or columns
             exclude_columns = exclude_columns or []
 
-            if (column not in include_columns) or (column in exclude_columns):
-                result.append("{} .. {}".format(actual, other))
+            if (
+                (column not in include_columns)
+                or (column in exclude_columns)
+                or matched is None
```

Comment:
yeah I remembered that you have asked me about this question, if column is not in "include columns" or in "exclude columns", the column MUST be a display only column, why still need to check that `matched` is `None`. As I read the code again, because here the `include_column` and `exclude_column` are totally not the same thing as that in `result.table.match`, when rendering the assertions, the data we have is a list of `RowComparision`, which is the result after matching, but it does not contain the original conditions such as `include_column` and `exclude_column`.

By the way, to keep the output much similar as web UI, I just use "" instead of "ABSENT" for empty cell.